cddl/cose-key.cddl

```diff
@@ -0,0 +1,10 @@
+COSE_KeySet = [ + COSE_Key ]
```

Comment:
I don't see yet why `COSE_KeySet` should be supported.

Reply:
How about adding the comments in the COSE_Key definition in [RFC8152] (https://datatracker.ietf.org/doc/html/rfc8152#section-7) to here?